_components/identifier/accessibility-tests.md

```diff
@@ -0,0 +1,11 @@
+---
+permalink: /components/identifier/accessibility-tests/
+layout: accessibility-test
+component:
+  name: Identifier
```

Comment:
**quibble:** It looks like the other test pages make this `name` value lower case.

```suggestion
  name: identifier
```